### PR TITLE
Static list

### PR DESCRIPTION
**Pull request by bensle**

### CODE REVIEW

**Review by smilenow24 — COMMENTED** [review]
Great Work! ;)

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,113 @@
-function App() {
-  return <div></div>;
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Heading from './components/Heading/Heading';
+
+const initialItems = [
```
Comment (macfrei) [inline src/App.js:5]: Once you have fetched the data, it is not really necessary to have a initial state anymore :)

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,113 @@
...
+  useEffect(() => {
+    loadShoppingItems();
+    async function loadShoppingItems() {
+      try {
+        const response = await fetch(
+          'https://fetch-me.vercel.app/api/shopping/items'
+        );
+        const data = await response.json();
+        setShoppingItems(data.data);
+        console.log(data);
```
Comment (macfrei) [inline src/App.js:73]: I would remove the `console.log` before merging to main. The main branch should always be as clean as possible :)

**Review by macfrei — COMMENTED** [review]
Nice job! :)

**Pull request merged**